fix: remove dynamic client registration (#1479)
defenseunicorns/uds-core: src/pepr/operator/README.md

          bearer_only: clientField(bearerOnly)
```

### How UDS Operator interacts with Keycloak

> The UDS Operator authenticates with Keycloak using the OAuth 2.0 Client Credentials Grant. It uses the OAuth 2.0 Client Credentials Grant to authenticate against the `uds-operator` client defined in Keycloak. This special client has a limited control over managing Keycloak Clients for the UDS Operator.

Reviewer: ```suggestion
The UDS Operator authenticates with Keycloak using the OAuth 2.0 Client Credentials Grant. It authenticates against the `uds-operator` client defined in Keycloak, which has limited control to manage Keycloak Clients for UDS.
```

This wording felt weird/redundant. Not sure if my suggestion is better, just tried to simplify a bit?
